app/Models/Post.php

```diff
@@ -366,7 +366,7 @@ public function toCustomerIoPayload()
                 'campaign_id' => (string) $this->campaign_id,
                 'campaign_title' => Arr::get($campaignWebsite, 'title'),
                 'campaign_slug' => Arr::get($campaignWebsite, 'slug'),
-                'campaign_cause' => implode(',', $campaign->cause),
+                'campaign_cause' => implode(',', $campaign->cause ?: []),
```

Comment:
For context, here's the error we were getting on this before (when`$campaign->cause` was `null`):

```
production.ERROR: implode(): Invalid arguments passed
```